components/brave_wallet_ui/components/extension/origin_info_card/origin_info_card.tsx

```diff
@@ -20,27 +27,41 @@ import {
 } from './origin_info_card.style'
 import { Column } from '../../shared/style'
 
+const isStorybook = isComponentInStorybook()
+
 interface Props {
   origin: BraveWallet.OriginInfo
 }
 
 export const OriginInfoCard = (props: Props) => {
   const { origin } = props
 
+  // Hooks
+  const { isDAppVerified, dapp } = useIsDAppVerified(origin)
+
+  // Computed
+  const dappIcon = dapp
+    ? isStorybook
+      ? dapp.logo
+      : `chrome://image?url=${encodeURIComponent(dapp.logo)}&staticEncode=true`
```

Comment:
nice, I forgot that the staticEncode defaults this to [always render it as a PNG](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/webui/sanitized_image_source.h;l=44-46?q=chrome:%2F%2Fimage&ss=chromium). This eliminates the issue I was originally concerned about of tampering/sanitization of the image on the filesystem.